lib/MusicBrainz/Server/WebService/Validator.pm

```diff
@@ -207,7 +219,13 @@ sub validate_inc
     my %extra;
     for my $i (@inc)
     {
-        map { $extra{$_} = 1 } @{ $extra_inc{$i} } if (defined $extra_inc{$i});
+        if (defined $extra_inc{$i}) {
+            my @available_extra_incs = (
+                @{ $extra_inc{$i}{generic} // [] },
+                @{ $extra_inc{$i}{$resource} // [] },
+            );
+            map { $extra{$_} = 1 } @available_extra_incs;
```

Comment:
Since you're already changing it,`for` is a lot clearer than using a `map` for side effects (TBH I thought you added a perlcritic rule for that already).

Reply:
I think you're thinking of https://github.com/metabrainz/musicbrainz-server/pull/2715